discord/channel.py

```diff
@@ -609,7 +615,8 @@ def type(self):
 
     def _update(self, guild, data):
         self.guild = guild
-        self.name = data['name']
+        self.name = data.get('name')
+        self.rtc_region = data['rtc_region']
```

Comment:
```suggestion
        self.name = data['name']
        self.rtc_region = data.get('rtc_region')
        if self.rtc_region:
            self.rtc_region = try_enum(VoiceRegion, self.rtc_region)
```

This probably needs to be imported. I don't know why the `name` key went from regular `__getitem__` to `get`.